### PR TITLE
New LLM page to the ML guide

### PR DESCRIPTION
**Pull request by mvsjober**
## Proposed changes

Add new LLM page to the machine learning guide. Added Accelerate and FSDP example to the Multi-GPU, multi-node page.

Previews:
- [New LLM page](https://csc-guide-preview.rahtiapp.fi/origin/llm-finetuning-examples/support/tutorials/ml-llm/)
- [Accelerate examples](https://csc-guide-preview.rahtiapp.fi/origin/llm-finetuning-examples/support/tutorials/ml-multi/#accelerate)
- [News item](https://csc-guide-preview.rahtiapp.fi/origin/llm-finetuning-examples/support/wn/training-new/#new-guide-for-working-with-large-language-models-on-csc-s-supercomputers-692024)
## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/support/tutorials/ml-multi.md [hunk 1]
```diff
@@ -674,6 +674,195 @@ PyTorch Lightning Slurm script for two full nodes using all GPUs:
...
+    ```
+
+=== "Mahti"
+
+    ```bash
+    #!/bin/bash
+    #SBATCH --account=<project>
+    #SBATCH --partition=gpumedium
+    #SBATCH --ntasks=1
+    #SBATCH --cpus-per-task=128
```
Comment (lupreCSC) [inline docs/support/tutorials/ml-multi.md:723]:
```suggestion
    #SBATCH --cpus-per-task=128
    #SBATCH --mem=510G
```

File: docs/support/tutorials/ml-multi.md [hunk 2]
```diff
@@ -674,6 +674,195 @@ PyTorch Lightning Slurm script for two full nodes using all GPUs:
...
+
+=== "Mahti"
+
+    ```bash
+    #!/bin/bash
+    #SBATCH --account=<project>
+    #SBATCH --partition=gpumedium
+    #SBATCH --nodes=2
+    #SBATCH --ntasks-per-node=1
+    #SBATCH --cpus-per-task=128
```
Comment (lupreCSC) [inline docs/support/tutorials/ml-multi.md:780]:
```suggestion
    #SBATCH --cpus-per-task=128
    #SBATCH --mem=510G
```

**Review by lupreCSC — CHANGES_REQUESTED** [review]
Looks good to me. Haven't had time to actually test the linked examples, though.

Removed one superfluous comma. I also noticed that Puhti batch scripts for the Accelerate section on Multi-GPU page specifies `--mem` while Mahti scripts don't, which seems a bit inconsistent. I added edit suggestions to fix that, please check if those are good.

File: docs/support/tutorials/ml-multi.md [hunk 3]
```diff
@@ -674,6 +674,195 @@ PyTorch Lightning Slurm script for two full nodes using all GPUs:
...
+A working [example for LLM fine-tuning can be found in this GitHub
+repository](https://github.com/CSCfi/llm-fine-tuning-examples)
+(check the files ending with `-accelerate.sh`). Also check our [guide
+on using LLMs on supercomputers](ml-llm.md).
+
+Example using Accelerate on all GPUs on a single node:
+
+=== "Puhti"
+
+    ```bash
```
[the remark this answers is not in the document]
Comment (mvsjober) [reply docs/support/tutorials/ml-multi.md:695]: Actually there's a good reason for this. On Mahti you get automatically 122.5GB of CPU memory per GPU allocated so these (and most other examples) do not include a memory allocation.

[the remark this answers is not in the document]
Comment (lupreCSC) [reply docs/support/tutorials/ml-multi.md:695]: I see, then let's keep it as it is

**Review by lupreCSC — APPROVED** [review]

**Pull request merged**